core/src/autogluon/core/_setup_utils.py

```diff
@@ -30,6 +30,8 @@
     "lightning": ">=2.1,<2.2",  # Major version cap
     "pytorch_lightning": ">=2.1,<2.2",  # Major version cap, capping `lightning` does not cap `pytorch_lightning`!
     "async_timeout": ">=4.0,<5",  # Major version cap
+    "transformers[sentencepiece]": ">=4.36.0,<4.39.0",
```

Comment:
Why do we use 4.36 instead of 4.38 as the lower cap?

Reply:
This is the lowest required by optimum for now. Wanted to leave it wide for more flexibility.